pip_audit/_cli.py

```diff
@@ -401,6 +410,10 @@ def audit() -> None:
             else:
                 spec = cast(ResolvedDependency, spec)
                 state.update_state(f"Auditing {spec.name} ({spec.version})")
+            if args.ignore_vulns:
+                filtered_vulns = [v for v in vulns if not v.has_any_id(set(args.ignore_vulns))]
```

Comment:
Tiny nit, but let's maybe memoize `set(args.ignore_vulns)` somewhere outside of the loop? That way we're not reallocating it on each audit step.